### ISSUE TITLE
To do: Enforce numbers greater than zero for attributes

### ISSUE THREAD

**Issue by CommanderTso** [CommanderTso/sinless-foundry-system#6]
It'd be nice to force character attributes to be a number of 1 or greater.  Currently not sure the right spot in the foundry logic cycle to do that, though.

**Comment by wyrmisis** [comment]
It'll take some refactoring, but you'd do this in a DataModel (which would help you shift some stuff out of the document files, and into per-document-subtype DataModels).

If this system still sees development, I'd be happy to chip in on this once I clear some other projects off of my plate.

**Comment by bacchist** [comment]
> It'll take some refactoring, but you'd do this in a DataModel (which would help you shift some stuff out of the document files, and into per-document-subtype DataModels).
> 
> If this system still sees development, I'd be happy to chip in on this once I clear some other projects off of my plate.

Development has been on hold for a bit, but we do intend on picking it back up in the near future. Any help would be greatly appreciated. Feel free to visit us on [Discord](https://discord.gg/UYzGqN8egQ) to chat about it. I'd love to pick your brain a bit, if nothing else.

**Comment by CommanderTso** [comment]
Yeah - thanks, Tim. I've had some priority shifts that have let me to
suspend active development, but Bacchist is still involved and running the
show for Sinless related development.
